docs/tutorials/vertical-fl.md

```diff
@@ -0,0 +1,113 @@
+# Cross-silo vertical federated learning
+
+## Background
+Vertical federated learning (VFL) is branch of federated learning where the data are split across the features among the participants rather than across the samples (horizontal FL). In other words we can say that it takes federated learning to another level as it allows for cross-organization collaboration without need for having the same features while keeping privacy and security of each individual's data intact. Some of real-world examples include, but are not limited to:
+- Finance: financial institution owning different pieces of data about their clients (e.g. bank account data, credit card data, loans data, ...etc)
```

Comment:
minor: this example was a bit confusing to me; it would maybe be more clear with _several_ institutions